intelmq/bin/intelmqctl.py

```diff
@@ -1219,11 +1221,11 @@ def botnet_status(self, group=None):
                 retval = 1
         return retval, botnet_status
 
-    def list(self, kind=None, non_zero=False, count=False):
+    def list(self, kind=None, non_zero=False, count=False, configured=False):
```

Comment:
The list of parameters for the `list` command gets a bit clunky and confusing now. I think we should split the `list` subcommand (maybe not part of this step, but soon)